### ISSUE TITLE
5.6.0

### ISSUE THREAD

**Issue by mmarousez-dc** [Webklex/php-imap#498]
Hello, I see that you have updates since 5.5.0 can you publish a new tag please ?

**Comment by theMeiner** [comment]
Yes pls. I need the new Version from nesbot/carbon to use the latest symfony parts

**Comment by wagnerbugs** [comment]
Yes. nesbot/carbon 3.6 (laravel 11)

**Comment by RootKeek** [comment]
Same here to use the latest symfony parts

**Comment by marien-probesys** [comment]
Note that, **as a temporary solution,** you can force Composer to use a newer version by specifying to use the `master` branch and pin the last commit:

```json
"require": {
    "webklex/php-imap": "dev-master#6d999438d29ed0bb920cd897b200a3a5fd6b6380"
}
```

This allows you to use the latest version that uses Carbon 3 (ref https://github.com/Webklex/php-imap/issues/483) and allows you to upgrade to Symfony 7 if you need to.

**You must be very careful with this!** As you pin the version number, you will no longer get updates from patch/minor versions. You will not even get information about new version when running `composer outdated`. I strongly recommend you to configure a notification system to know when a new release is available (e.g. by following the RSS feed of the releases https://github.com/Webklex/php-imap/releases.atom)

**Comment by theMeiner** [comment]
But if you have required roave/security-advisories
You will receive this

`Problem 1
    - Root composer.json requires webklex/php-imap dev-master -> satisfiable by webklex/php-imap[dev-master].
    - roave/security-advisories dev-latest conflicts with webklex/php-imap 1.0.x-dev.
    - webklex/php-imap 1.0.x-dev is an alias of webklex/php-imap dev-master and must be installed with it.
    - roave/security-advisories is locked to version dev-latest and an update of this package was not requested.
`

Because of you branch-alias is set to version 1.0.x-dev*

When i try to req with version 1.0.x-dev, because is an alias, i get this

` Problem 1
    - Root composer.json requires webklex/php-imap 1.0.x-dev -> satisfiable by webklex/php-imap[1.0.x-dev (alias of dev-master)].
    - roave/security-advisories dev-latest conflicts with webklex/php-imap <5.3.
    - Root composer.json requires roave/security-advisories dev-latest -> satisfiable by roave/security-advisories[dev-latest].
`

You can update the alias to 5.5.* to avoid this right?

**Comment by mariuszkrzaczkowski** [comment]
When will the next version of the change be published?

**Comment by netpok** [comment]
@Webklex Please create a release if possible

**Comment by netpok** [comment]
But https://github.com/Webklex/php-imap/issues/521 needs fixing first

**Comment by igordeveloper** [comment]
> Observe que, **como solução temporária,** você pode forçar o Composer a usar uma versão mais recente especificando o uso da `master`ramificação e fixando o último commit:
> 
> ```json
> "require": {
>     "webklex/php-imap": "dev-master#6d999438d29ed0bb920cd897b200a3a5fd6b6380"
> }
> ```
> 
> Isso permite que você use a versão mais recente que usa o Carbon 3 (ref #483 ) e permite que você atualize para o Symfony 7 se necessário.
> 
> **Você deve ter muito cuidado com isso!** Ao fixar o número da versão, você não receberá mais atualizações de versões de patch/menores. Você nem mesmo obterá informações sobre a nova versão ao executar `composer outdated`. Eu recomendo fortemente que você configure um sistema de notificação para saber quando uma nova versão estiver disponível (por exemplo, seguindo o feed RSS das versões https://github.com/Webklex/php-imap/releases.atom )
> 
> Nota lateral: se você usa o GitHub, pode receber um alerta de segurança do Dependabot sobre #416 . O relatório está incorreto, pois você usa uma versão do PHP-IMAP > 5.3. Você pode descartá-lo.



Hello, 

With this soltion i have a problem with roave/security-advisories, see

   - roave/security-advisories dev-master conflicts with webklex/php-imap <5.3.

could you please push a new tag release, for roave/security-advisories >= 5.4 its ok


thank you.

**Comment by igordeveloper** [comment]
> > Observe que, **como solução temporária,** você pode forçar o Composer a usar uma versão mais recente especificando o uso da `master`ramificação e fixando o último commit:
> > ```json
> > "require": {
> >     "webklex/php-imap": "dev-master#6d999438d29ed0bb920cd897b200a3a5fd6b6380"
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Isso permite que você use a versão mais recente que usa o Carbon 3 (ref #483 ) e permite que você atualize para o Symfony 7 se necessário.
> > **Você deve ter muito cuidado com isso!** Ao fixar o número da versão, você não receberá mais atualizações de versões de patch/menores. Você nem mesmo obterá informações sobre a nova versão ao executar `composer outdated`. Eu recomendo fortemente que você configure um sistema de notificação para saber quando uma nova versão estiver disponível (por exemplo, seguindo o feed RSS das versões https://github.com/Webklex/php-imap/releases.atom )
> > Nota lateral: se você usa o GitHub, pode receber um alerta de segurança do Dependabot sobre #416 . O relatório está incorreto, pois você usa uma versão do PHP-IMAP > 5.3. Você pode descartá-lo.
> 
> Hello,
> 
> With this soltion i have a problem with roave/security-advisories, see
> 
> * roave/security-advisories dev-master conflicts with webklex/php-imap <5.3.
> 
> could you please push a new tag release, for roave/security-advisories >= 5.4 its ok
> 
> thank you.

@marien-probesys 

Is there any point blocking a new release with this PR?

**Comment by igordeveloper** [comment]
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires webklex/php-imap dev-master#6d999438d29ed0bb920cd897b200a3a5fd6b6380 -> satisfiable by webklex/php-imap[dev-master].
    - roave/security-advisories dev-master conflicts with webklex/php-imap <5.3.
    - webklex/php-imap 1.0.x-dev is an alias of webklex/php-imap dev-master and must be installed with it.
    - Root composer.json requires roave/security-advisories dev-master -> satisfiable by roave/security-advisories[dev-master].

@marien-probesys

**Comment by igordeveloper** [comment]
> Note that, **as a temporary solution,** you can force Composer to use a newer version by specifying to use the `master` branch and pin the last commit:
> 
> ```json
> "require": {
>     "webklex/php-imap": "dev-master#6d999438d29ed0bb920cd897b200a3a5fd6b6380"
> }
> ```
> 
> This allows you to use the latest version that uses Carbon 3 (ref #483) and allows you to upgrade to Symfony 7 if you need to.
> 
> **You must be very careful with this!** As you pin the version number, you will no longer get updates from patch/minor versions. You will not even get information about new version when running `composer outdated`. I strongly recommend you to configure a notification system to know when a new release is available (e.g. by following the RSS feed of the releases https://github.com/Webklex/php-imap/releases.atom)
> 
> Side note: if you use GitHub, you may receive a security alert from Dependabot about #416. The report is incorrect as you use a version of PHP-IMAP > 5.3. You can dismiss it.

Hello @marien-probesys 

With this solution i have a problem with other bundle, the  roave/security-advisories, see error:

roave/security-advisories dev-master conflicts with webklex/php-imap <5.3.

Could you please push a new release or help me with a solution to use this commit with roave/security-advisories together?

Thank you!!!

**Comment by igordeveloper** [comment]
> But #521 needs fixing first

this was fixed?

**Comment by MrJuliuss** [comment]
@Webklex Hi, any news about 5.6.0 ? 5.5 does not work on Symfony 7+ because of Carbon 2 :/

**Comment by marien-probesys** [comment]
For information, PHP-IMAP 6.0 has been released: https://github.com/Webklex/php-imap/releases/tag/6.0.0

**Comment by Webklex** [comment]
Hi @mmarousez-dc @theMeiner @wagnerbugs @RootKeek @mariuszkrzaczkowski @netpok @igordeveloper @MrJuliuss ,
a new release is available, however there are already more changes in the pipeline :)

Thanks for your patience & happy coding,
